ako-gateway-api/k8s/gateway_controller.go

```diff
@@ -103,6 +110,86 @@ func (c *GatewayController) SetupEventHandlers(k8sinfo k8s.K8sinformers) {
 	c.workqueue = mcpQueue.Workqueue
 	numWorkers := mcpQueue.NumWorkers
 
+	// Add EPSInformer
+	epsEventHandler := cache.ResourceEventHandlerFuncs{
+		AddFunc: func(obj interface{}) {
+			if c.DisableSync {
+				return
+			}
+			eps := obj.(*discovery.EndpointSlice)
+			namespace, _, _ := cache.SplitMetaNamespaceKey(utils.ObjKey(eps))
+			svcName, ok := eps.Labels[discovery.LabelServiceName]
+			if !ok || svcName == "" {
+				utils.AviLog.Debugf("Endpointslice Add event: Endpointslice does not have backing svc")
+				return
+			}
+			key := utils.Endpointslices + "/" + namespace + "/" + svcName
+			if lib.IsNamespaceBlocked(namespace) {
```

Comment:
Not very sure whether NamespaceBlocked is supported by gateway or not.
Can any of @aaha97 or @akshayhavile confirm this ?

Reply:
removed